lib/activerecord-import/import.rb

```diff
@@ -454,7 +450,7 @@ def find_associated_objects_for_import(associated_objects_by_class, model)
         changed_objects.each do |child|
           child.send("#{association_reflection.foreign_key}=", model.id)
         end
-        associated_objects_by_class[model.class.name][association_reflection.name].concat changed_objects
+        associated_objects_by_class[model.class.name].concat changed_objects
       end
```

Comment:
This is a bug, isn't it? What if you have two associations, both with the same class name, but named differently. Ie, 

    class Student
        has_many :required_courses, :class_name=>'Course'
        has_many :optional_courses, :class_name=>'Course'